### PR TITLE
osdc/objecter: fix compile error

### PR DESCRIPTION
**Pull request by hjwsm1989**
Signed-off-by: huangjun <huangjun@xsky.com>

### CODE REVIEW

File: src/osdc/Objecter.cc [hunk 1]
```diff
@@ -4235,7 +4235,7 @@ void Objecter::_finish_pool_stat_op(PoolStatOp *op, int r)
   delete op;
 }
 
-void Objecter::get_fs_stats(ceph_statfs& result,
+void Objecter::get_fs_stats(struct ceph_statfs& result,
```
Comment (tchaikov) [inline src/osdc/Objecter.cc]: what compiling error are you running into? i am able to "make osdc" just fine locally.

Comment (amitkumar50) [reply src/osdc/Objecter.cc]: @hjwsm1989 Agree with kefu.
Just do recent clone. build it. `make osdc` complies fine without your change.

**Comment by liewegas** [comment]
builds for me too?

File: src/osdc/Objecter.cc (hunk 1 above)
Comment (hjwsm1989) [reply src/osdc/Objecter.cc]: @tchaikov @amitkumar50 I don't remember what error now, so just close it.

**Pull request closed without merging**